clang/lib/Driver/Driver.cpp

```diff
@@ -4158,18 +4158,14 @@ class OffloadingActionBuilder final {
       // supported GPUs.  sm_20 code should work correctly, if
       // suboptimally, on all newer GPUs.
       if (GpuArchList.empty()) {
-<<<<<<< HEAD
-        if (ToolChains.front()->getTriple().isSPIROrSPIRV())
-          GpuArchList.push_back(CudaArch::Generic);
-        else
-=======
         if (ToolChains.front()->getTriple().isSPIRV()) {
```

Comment:
Use `isSPIRorSPIRV()` here
```suggestion
        if (ToolChains.front()->getTriple().isSPIRorSPIRV()) {
```